docs/pacific/New-Caledonia/TCU.md

```diff
@@ -0,0 +1,64 @@
+---
+  title: Tontouta TCU
+---
+
+--8<-- "includes/abbreviations.md"
+
+## Positions
+
+| Name               | ID      | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | --------------| -------------- | ---------------- | --------------------------------------|
+| Tontouta Approach South|NWWWA| Tontouta Approach  | 128.300         | NWWW_APP          |
+
+## Airspace
+
+The vertical limits of the Tontouta TCU are `SFC` to `F245`.
+
+When NWWM ADC is offline, the NWWM CTR (`SFC` to `A015`) reverts to Class G and is administered by the relevant TCU controller.
+
+## Coordination
+### NWWW TCU / ENR
+
+#### Departures
+
+Voiceless coordination is in place from NWWW TCU to NWWW/NFFF for aircraft assigned the lower of `F240` or the `RFL` and tracking via a procedural SID terminus.
+
+Any aircraft not meeting the above requirements must be prior coordinated to NWWW/NFFF.
+
+#### Arrivals
+
+Voiceless coordination is in place from NWWW/NFFF to NWWW TCU given the arrival has been assigned the standard assignable level of `A100` and depending on the duty runway:
+
+|Runway|Requirement|
+|------|-----------|
+|11|Assigned a STAR|
+|29|Tracking to LTO VOR|
+
+All other aircraft must be prior coordinated.
+
+### NWWW TCU / NWWW ADC
+#### Auto Release
+
+'Next' coordination is **not** required to from NWWW ADC for aircraft that are:
+
+- Departing from a runway nominated on the ATIS; and  
+- Assigned the standard assignable level; and  
+- Assigned a **Procedural** SID
+
+### NWWW TCU / NWWM ADC
+#### Departures
+
+When the aircraft is ready for departure, NWWM ADC will coordinate with NWWW TCU for permission to release the aircraft into CTA.
+
+The standard assignable level from NWWM ADC to NWWW TCU is the lower of `A050` or the `RFL`, any other level must be prior coordinated.
+
+#### Arrivals/Overfliers
+NWWW TCU will heads-up coordinate arrivals/overfliers from Class D to NWWM ADC prior to **5 mins** from the boundary.
```

Comment:
Could we please add an example for this? Anything more or less copy and pasted from an AUS class D AD will be fine

Reply:
Most of this is already copypasted from AU SOP’s.

In regard to the ENR handoffs, from talking to an SME from New Caledonia they utilise ADS-B for surveillance and 25nm is their standard due to airspace size constraints. As such I’d rather stick with what they do irl instead of using the default AU SOP.